src/htmlgen.cpp

```diff
@@ -3475,7 +3475,7 @@ void HtmlGenerator::writeInheritedSectionTitle(
   DBG_HTML(m_t << "<!-- writeInheritedSectionTitle -->\n";)
   QCString a = anchor;
   if (!a.isEmpty()) a.prepend("#");
-  QCString classLink = QCString("<a class=\"el\" ");
+  QCString classLink = QCString("<a class=\"RENAMED4_el\" ");
```

Comment:
Probably based on some old and incorrect changes for some other issues you had, to be reverted